composer.json

```diff
@@ -17,7 +17,7 @@
   ],
   "require": {
     "php": "^5.6 || ^7.0",
-    "illuminate/support": "5.3.* || 5.4.* || 5.5.* || 5.6.*"
+    "illuminate/support": "5.3.* || 5.4.* || 5.5.* || 5.6.* || 5.7.*"
```

Comment:
👍

Reply:
what is the problem with add support to laravel 5.7?

Reply:
No problem!